openlibrary/core/schema.sql

```diff
@@ -49,3 +49,232 @@ CREATE TABLE bookshelves_votes (
 INSERT INTO bookshelves (name, description) VALUES ('Want to Read', 'A list of books I want to read');
 INSERT INTO bookshelves (name, description) VALUES ('Currently Reading', 'A list of books I am currently reading');
 INSERT INTO bookshelves (name, description) VALUES ('Already Read', 'A list of books I have finished reading');
+
+
+CREATE TABLE observation_types (
+    id serial not null primary key,
+    type text,
+    description text,
+    allow_multiple_values BOOLEAN,
+    created timestamp without time zone default (current_timestamp at time zone 'utc'),
+    updated timestamp without time zone default (current_timestamp at time zone 'utc')
+);
+
+-- New prev_value must be set before a row is deleted
+CREATE TABLE observation_values (
```

Comment:
Add documentation describing how `prev_value` is used to order values.  A short example would be ideal.